<launch>
  <param name="use_sim_time" value="true"/>

	<node pkg="rviz" type="rviz" name="rviz"
		args="-d $(find ar_pose)/demo/demo_multi.vcg"/>

  <node pkg="tf" type="static_transform_publisher" name="world_to_cam" 
    args="0 0 0.5 -1.57 0 -1.57 world usb_cam 100" />

	<node name="ar_pose" pkg="ar_pose" type="ar_multi" respawn="false" output="screen">
		<param name="marker_pattern_list" type="string" value="data/object_4x4"/>
		<param name="threshold" type="int" value="100"/>
	</node>
	
	<node pkg="rosbag" type="rosbag" name="rosbag" 
		args="play $(find ar_pose)/demo/usb_cam_topics_ar_multi.bag --clock -d 3"/>
	
</launch>
